src/providers/liveupdates/BasicPubSubManager.hpp

```diff
@@ -337,12 +337,15 @@ class BasicPubSubManager
     WebsocketContextPtr onTLSInit(const websocketpp::connection_hdl & /*hdl*/)
     {
         WebsocketContextPtr ctx(
-            new boost::asio::ssl::context(boost::asio::ssl::context::tlsv12));
+            new boost::asio::ssl::context(boost::asio::ssl::context::tlsv13));
 
         try
         {
             ctx->set_options(boost::asio::ssl::context::default_workarounds |
                              boost::asio::ssl::context::no_sslv2 |
+                             boost::asio::ssl::context::no_sslv3 |
```

Comment:
No need to disable SSLv2/3 because `tls_client` [implies TLS 1+](https://github.com/boostorg/asio/blob/62864c570afcb7234442168ecacd53af701d7d9e/include/boost/asio/ssl/impl/context.ipp#L334).